.rubocop.yml

```diff
@@ -1,63 +1,72 @@
+
+# List of cops can be found here:
+# https://github.com/bbatsov/rubocop/blob/master/config/default.yml
+# https://github.com/bbatsov/rubocop/blob/master/config/disabled.yml
+# https://github.com/bbatsov/rubocop/blob/master/config/enabled.yml
+
 inherit_from: .rubocop_todo.yml
 
 AllCops:
   DisplayCopNames: true
-  Include:
-    - '**/Rakefile'
-    - '**/config.ru'
+  DisplayStyleGuide: true
   Exclude:
     - 'db/**/*'
-    - 'script/**/*'
-    - 'spec/internal/**/*'
-    - 'spec/test_app_templates/**/*'
```

Comment:
Removes vendor directory by default. The rest of these, such as spec/internal and test_app_templates are bad copy paste from sufia.